index.md

```diff
@@ -14,298 +34,27 @@ A more refined and detailed version of the EBML specifications is being [worked
-| Global elements (used everywhere in the format) |
-| <a name="Void"></a>Void | g | [EC] | - | - | - | - | <abbr title="Binary">b</abbr> | * | * | * | * | * | Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use. |
-| <a name="CRC-32"></a>CRC-32 | g | [BF] | - | - | - | - | <abbr title="Binary">b</abbr> | * | * | * | * | The CRC is computed on all the data of the Master element it's in. The CRC element should be the first in it's parent master for easier reading. All level 1 elements should include a CRC-32\. The CRC in use is the IEEE CRC32 Little Endian |
-| <a name="SignatureSlot"></a>SignatureSlot | g | [1B][53][86][67] | - | mult. | - | - | <abbr title="Master Elements">m</abbr> | Contain signature of some (coming) elements in the stream. |
-| <a name="SignatureAlgo"></a>SignatureAlgo | 1 | [7E][8A] | - | - | - | - | <abbr title="Unsigned Integer">u</abbr> | Signature algorithm used (1=RSA, 2=elliptic). |
-| <a name="SignatureHash"></a>SignatureHash | 1 | [7E][9A] | - | - | - | - | <abbr title="Unsigned Integer">u</abbr> | Hash algorithm used (1=SHA1-160, 2=MD5). |
-| <a name="SignaturePublicKey"></a>SignaturePublicKey | 1 | [7E][A5] | - | - | - | - | <abbr title="Binary">b</abbr> | The public key to use with the algorithm (in the case of a PKI-based signature). |
-| <a name="Signature"></a>Signature | 1 | [7E][B5] | - | - | - | - | <abbr title="Binary">b</abbr> | The signature of the data (until a new. |
-| <a name="SignatureElements"></a>SignatureElements | 1 | [7E][5B] | - | - | - | - | <abbr title="Master Elements">m</abbr> | Contains elements that will be used to compute the signature. |
-| <a name="SignatureElementList"></a>SignatureElementList | 2 | [7E][7B] | - | mult. | - | - | <abbr title="Master Elements">m</abbr> | A list consists of a number of consecutive elements that represent one case where data is used in signature. Ex: _Cluster|Block|BlockAdditional_ means that the BlockAdditional of all Blocks in all Clusters is used for encryption. |
-| <a name="SignedElement"></a>SignedElement | 3 | [65][32] | - | mult. | - | - | <abbr title="Binary">b</abbr> | An element ID whose data will be used to compute the signature. |
-
-The default values defined for the EBML header correspond to the values for a Matroska stream/file. When parsing the EBML header the default values are different, irrespective of the DocType defined.
-
-*   EBMLMaxIDLength is 4: IDs in the EBML header cannot be longer than 4 octets.
-*   EBMLMaxSizeLength is 4: Length of IDs in the EBML header cannot be longer than 4 octets.
```

Comment:
I think this line of the spec was incorrect, should be 8 or less, corrected in https://github.com/Matroska-Org/matroska-specification/pull/30/commits/1a7943bf0072a6a82fb7b9fd97803e2d60c9f4ed